### PR TITLE
desktop browser promo message for iOS and Android

### PR DESCRIPTION
**Pull request by brindy**
**Requests**

* Android: https://app.asana.com/0/0/1205395518207473/f
* iOS: https://app.asana.com/0/0/1205395610200832/f 

**Merge Schedule**

* Monday 25 Sep, AM

**Screenshots**

image2
Simulator Screenshot - iPhone 14 Pro Max - 2023-09-21 at 15 14 21
Simulator Screenshot - iPhone 14 Pro Max - 2023-09-21 at 15 14 14

### CODE REVIEW

File: live/ios-config/ios-config.json [hunk 1]
```diff
@@ -1,5 +1,35 @@
 {
-  "version": 6,
-  "messages": [],
-  "rules": []
+  "version": 7,
+  "messages": [
+    {
+      "id": "macos_promo_may2023",
```
Comment (brindy) [inline live/ios-config/ios-config.json:5]: This is so that people who saw this during the initial experiment don't see it again.

**Comment by cmonfortep** [comment]
thanks @brindy 

I see the config is pushed to staging for Android:
https://staticcdn.duckduckgo.com/remotemessaging/config/staging/android-config.json

It seems on iOS it has been pushed to:
https://staticcdn.duckduckgo.com/remotemessaging/config/staging/pre/ios-config.json

@amddg44 do you know why url has "pre" in the path? is that expected?

**Review by cmonfortep — APPROVED** [review]
LGTM

File: live/android-config/android-config.json [hunk 2]
```diff
@@ -1,5 +1,37 @@
 {
-  "version": 4,
-  "messages": [],
-  "rules": []
-}
\ No newline at end of file
+  "version": 5,
```
Comment (cmonfortep) [inline live/android-config/android-config.json:2]: @nalcalag if you want to double check, here's the config

**Comment by amddg44** [comment]
> thanks @brindy
> 
> I see the config is pushed to staging for Android: https://staticcdn.duckduckgo.com/remotemessaging/config/staging/android-config.json
> 
> It seems on iOS it has been pushed to: https://staticcdn.duckduckgo.com/remotemessaging/config/staging/pre/ios-config.json
> 
> @amddg44 do you know why url has "pre" in the path? is that expected?
> 
> EDIT: ok I think it is expected https://github.com/duckduckgo/remote-messaging-config/blob/main/.github/workflows/ios-PR-to-staging.yml#L32 all good 👍

Ah yes 🤦  When the first remote message was going out I don't know if you remember but I discovered I'd accidentally set the wrong url in the app - it was set to staging instead of production so I had to modify the flow slightly so that PRs wouldn't trigger remote messages to start appearing for users.

File: live/ios-config/ios-config.json [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+      "content": {
+        "messageType": "promo_single_action",
+        "titleText": "Did you know our free browser is available on Windows & Mac?",
+        "descriptionText": "Visit <b>duckduckgo.com/browser</b> on your computer to download.",
+        "placeholder": "NewForMacAndWindows",
+        "actionText": "Send Link",
+        "action": {
+          "type": "share",
+          "value": "Visit https://duckduckgo.com/browser",
```
Comment (nalcalag) [inline live/ios-config/ios-config.json:14]: Just double check. I think we agreed on adding _"Visit "_ before the link on Android to show the correct title for the sharesheet on Samsung devices but not needed on iOS?

**Pull request merged**